scripts/generate_docs.sc

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env amm
+
+import java.time.Instant
+import ammonite.ops._
+import ammonite.ops.ImplicitWd._
+
+/**
+  * Docs generation process:
+  *
+  * 1. copy docs/docs and docs/_layouts/docs.html part from 1.3, 1.4 and 1.5 branches to the temp dirs
+  * 2. copy the latest release (1.5_ branch doc to the temp dir
+  * 3. generate the latest release docs
+  * 4. replace docs/docs with docs/docs from 1.3, 1.4, 1.5 and generate docs for the _sites/docs_version
+  */
+
+def makeTmpDir(): Path = {
+  val timestamp = Instant.now().toString.replace(':', '-')
+  val path = root/"tmp"/s"marathon-docs-build-$timestamp"
+  mkdir! path
+  path
+}
+
+val latestReleaseVersion = "1.5"
```

Comment:
Hm. This would be another version to hardcode. I'd opt for using the version script from #5970.

Reply:
well, it's not exactly version, more a name of the release branch

Reply:
As written above, I'd go for tags instead.